app.py

```diff
@@ -2,36 +2,49 @@
 import binascii
 import io
 
-from flask import Flask, request, render_template, jsonify
+from flask import Flask, jsonify, render_template, request
 from werkzeug.exceptions import BadRequest
 
-import warn_old_config
```

Comment:
Why was this warning removed?
```suggestion
import warn_old_config
```